cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc

```diff
@@ -59,20 +59,13 @@ CREATE OR REPLACE USER name
 
 | [source]
 ----
-ALTER USER name SET {
-[PLAINTEXT \| ENCRYPTED] PASSWORD password
+ALTER USER name
+  [SET [PLAINTEXT \| ENCRYPTED] PASSWORD password]
   [[SET PASSWORD] CHANGE [NOT] REQUIRED]
```

Comment:
I think this would be a closer representation of the syntax
```suggestion
  [SET [PLAINTEXT \| ENCRYPTED] PASSWORD password [CHANGE [NOT] REQUIRED]]
  [SET PASSWORD CHANGE [NOT] REQUIRED]
```

Reply:
It does make it seem like you could do `SET PASSWORD 'password' CHANGE NOT REQUIRED SET PASSWORD CHANGE REQUIRED`, but maybe yes? Or maybe
```suggestion
  {[SET [PLAINTEXT \| ENCRYPTED] PASSWORD password [CHANGE [NOT] REQUIRED]] \|
  [SET PASSWORD CHANGE [NOT] REQUIRED]}
```

Reply:
Hmm, then I almost prefer it as in your first version, with the additional comment you had in the text explaining when the [SET PASSWORD] wasn't required

Reply:
So we leave it be then? :P